Sources/SwiftInspectorVisitors/TypeSyntax+NameExtractor.swift

```diff
@@ -0,0 +1,25 @@
+// Created by dan_federman on 1/30/21.
+// Copyright © 2021 Airbnb Inc. All rights reserved.
+
+import SwiftSyntax
+
+extension TypeSyntax {
+
+  /// Returns the the qualified name for the type the receiver represents.
+  var qualifiedName: String {
```

Comment:
Let's clarify in the documentation when it's safe to call this.

Reply:
good call. I may eventually want to factor this out to make it more compile-time safe.

2fcfd31a00eff9d58c21f9748d27fab349d8bb97